cobra/flux_analysis/loopless.py

```diff
@@ -96,8 +97,7 @@ def _add_cycle_free(model, fluxes):
             rxn.bounds = min(flux, rxn.lower_bound), min(0, rxn.upper_bound)
             objective_vars.append(rxn.reverse_variable)
 
-    model.objective.set_linear_coefficients(dict.fromkeys(objective_vars, 1.0))
-    model.objective.direction = "min"
+    model.objective.set_linear_coefficients({v: 1.0 for v in objective_vars})
```

Comment:
Was there a particular reason for that? I always thought that `dict.fromkeys` is actually faster than a dict comprehension...

Reply:
In my tests the dict comprehension was slightly faster (hardly worth mentioning). I did it more because we were already using comprehensions in other places and I find it a bit more intuitive.